app/src/main/java/com/github/factotum_sdp/factotum/ui/roadbook/RoadBookViewAdapter.kt

```diff
@@ -81,21 +82,27 @@ class RoadBookViewAdapter : RecyclerView.Adapter<RoadBookViewAdapter.RecordViewH
         if (date == null)
```

Comment:
From what I know it's used to have some code block executed if the object before the .? is non-null. Otherwise the code inside the {} is not executed.
But here, I want to set an explicit value ("_") to display on the screen when date is null, which means that no timestamp has been edited for the moment.

Reply:
However using the elvis operator : I can write like this :
 ```
date?.let {
            sb.append(SimpleDateFormat.getTimeInstance().format(it))
        } ?: sb.append('_').toString()
```